classic menu bar/css/style.css

```diff
@@ -0,0 +1,151 @@
+.wrapper {
+  position: relative;
+  background-image: url("../images/birds.jpg");
+  background-position: center;
+  width: 300px;
+  height: 580px;
+  border: grey solid;
+  box-shadow: var(0px 1px 5px rgba(104, 104, 104, 0.8));
+  border-radius: 15px;
+  margin: 0 auto;
+  overflow: hidden;
+}
+
+.menu-icon {
+  position: absolute;
+  bottom: 7px;
+  right: 7px;
+  cursor: pointer;
+}
+
+.icon-list-item {
+  display: flex;
+  position: absolute;
+  bottom: 55px;
+  right: 7px;
+}
+
+.icon-box {
+  width: 40px;
+  height: 40px;
+  border: #cecaca solid;
+  padding-right: 0;
+  border-radius: 50%;
+}
+
+.item {
+  display: flex;
+  width: 120px;
+  height: 60px;
+  position: absolute;
+  justify-content: space-between;
+}
+
+.icon {
+  color: #beb7b7;
+  position: absolute;
+  top: 11px;
+  left: 11px;
+}
+
+.icon-text {
+  direction: rtl;
+  text-align: right;
+  position: absolute;
+  width: 60px;
+  bottom: 7px;
+  color: #ffffff;
+  padding-top: 4px;
+}
+
+.icon-container {
+  position: absolute;
+  margin-right: 0px;
+  padding-right: 0px;
+  bottom: 6px;
+}
+
+.item.open:nth-child(1) {
+  position: absolute;
+  bottom: calc(55px*1);
+  transform: translateX(calc(-6*0*0px - 200px));
+  transition: transform 0.7s;
+}
```

Comment:
great work although alternatively, js could have lessened writing out everything.

Reply:
Yeah, you are right i did not use it because for this work i thought of utilizing powers of sass :smile: